src/lib/env.ts

```
@@ -1,14 +1,29 @@
 if (envPath && existsSync(envPath)) {
+  // 1. SHOOTER_HOME/.env
   config({ path: envPath });
 } else {
+  // 2. .env in cwd (dotenv default — only populates vars not already set)
   config();
+
+  // 3. ~/.shooter/.env as final fallback
+  const fallbackPath = join(homedir(), '.shooter', '.env');
+  if (existsSync(fallbackPath)) {
+    // override: false ensures cwd .env values take precedence
+    config({ path: fallbackPath, override: false });
```

ESLint

[error] Expected "override" to come before "path".

(perfectionist/sort-objects)